twitter.css

```diff
@@ -0,0 +1,209 @@
+
```

Comment:
My assumption is that the page is not fully complete yet. Here are few things that i noticed so far: 
1. Page is working with grid but left and right sides are not occupying correct grid to display properly. 
2. Use of icons for blue side is a good practice. 
Please continue to work on this assignment on your free time. It is important to continue to learn CSS to make your future projects look cool and user friendly. Please reach out to me if you need help or advice on making this page better. Keep it up! 👍